webapp/components/login/login.jsx

```diff
@@ -35,6 +35,8 @@ export default class Login extends React.Component {
         this.state = state;
     }
     componentDidMount() {
+        document.title = global.window.mm_config.SiteName;
```

Comment:
Maybe I don't understand how `document.title` works. I assumed if you set it it would stick until it was set again. So having it done once on comonentDidMount would be good enough. Is this not the case?